config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -290,7 +343,7 @@ presubmits:
             # Panic if data inconsistency is detected
             - --env=ENABLE_KUBE_LIST_FROM_CACHE_INCONSISTENCY_DETECTOR=true
             - --stage=gs://kubernetes-release-pull/ci/pull-kubernetes-e2e-gce
-            - --test_args=--ginkgo.skip=\[Driver:.gcepd\]|\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\] --minStartupPods=8
+            - --test_args=--ginkgo.skip=\[Driver:.gcepd\]|\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[NodeFeature:.+\] --minStartupPods=8
```

Comment:
what are the implications of this change?

Reply:
hmm, we loose coverage, there are already more than 20 tests running that will be skipped because of this

https://testgrid.k8s.io/google-gce#pull-kubernetes-e2e-gce&include-filter-by-regex=NodeFeature

Reply:
Good catch, updated regex to specifically skip `NodeFeature:GracefulNodeShutdown` (which is the goal here and safe to do as this job does not enable the graceful shutdown feature intentionally).